custom-generator/src/main/java/io/serverlessworkflow/generator/AllAnyOneOfSchemaRule.java

```diff
@@ -54,7 +59,35 @@ class AllAnyOneOfSchemaRule extends SchemaRule {
     this.ruleFactory = ruleFactory;
   }
 
-  private static class JTypeWrapper {
+  private static final String REF = "$ref";
+  private static final String PATTERN = "pattern";
+
+  private enum Format {
+    URI_TEMPLATE("^[A-Za-z][A-Za-z0-9+\\-.]*://.*");
```

Comment:
The idea is to add adittional formats when needed. 
Right now, the only format that is used by the spec and not handle is json-pointer, but I could not find a good one for this ABNF yet
```
  json-pointer    = *( "/" reference-token )
      reference-token = *( unescaped / escaped )
      unescaped       = %x00-2E / %x30-7D / %x7F-10FFFF
         ; %x2F ('/') and %x7E ('~') are excluded from 'unescaped'
      escaped         = "~" ( "0" / "1" )
        ; representing '~' and '/', respectively
```